### ISSUE TITLE
Logitech C615 microphone not recognized

### ISSUE THREAD

**Issue by pmaziere** [jitsi/jitsi-meet-electron#300]
Hello,

Using 2.0.2  appimage on debian testing, while the camera works as expected, the microphone is not recognized.

Everything works as expected when using Firefox on the same server.
The camera and microphone are correctly recognized and functional  with a tool like guvcview.

Moreover the same appimage works fine with the embbeded webcam of several debian laptops I was able to test it on.

Therefore I think it's not a configuration issue on my system, but rather something specific to my webcam originating from the appimage.

Thanks for trying to solve this issue

**Comment by saghul** [comment]
Does it work in Chromium? That's as close to this app as it gets.

**Comment by pmaziere** [comment]
sorry for the delayed reply.

It does not work in Chromium: the microphone can not be accessed

**Comment by saghul** [comment]
Then there is nothing we can do. Electron is basically Chromium with a very good disguise and some extras.

**Comment by susobaco** [comment]
I have a similar problem, I do not recognize the microphone via pulseaudio. In chromium I didn't detect it either, but I managed to get it to recognize it, by manipulating the configuration in chrome://settings/content/microphone .
I use debian 10 with version 2.3.0 jitsi-meet-electron. (appimage)

**Comment by Pat-Soal** [comment]
> I have a similar problem, I do not recognize the microphone via pulseaudio. In chromium I didn't detect it either, but I managed to get it to recognize it, by manipulating the configuration in chrome://settings/content/microphone .
> I use debian 10 with version 2.3.0 jitsi-meet-electron. (appimage)

I had the same issue and found a fix on debian buster

In Jitsi it would say permissions not granted for the mic in settings.
My mic was not detected in jitsi or in chromium until going into chromium settings and manually selecting the mic from a dropdown box deep in the apps settings. 

I fixed my issue by:

1. opening the pulse audio mixer (pavucontrol from terminal) << output devices tab << toggled  on the green checkbox (set as fallback device)

2. I went back into a random jitsi call and toggled the mic, it enabled this time. the mic is automatically detected and permissions are requested in chromium as well without having to dig into the settings

I hope this helps.
